tools/devtools/utils.mjs

```diff
@@ -0,0 +1,262 @@
+import {readFile, writeFile, mkdir} from 'fs/promises';
+// eslint-disable-next-line node/no-extraneous-import
+import nunjucks from 'nunjucks';
+import {Octokit} from 'octokit';
+import {dirname} from 'path';
+
+nunjucks.configure({
+  autoescape: false,
+  tags: {blockStart: '<%', blockEnd: '%>'},
```

Comment:
If you are wondering why I change the block syntax, it's because I don't want nunjucks to render `{% include }`, 11ty should handle it.

Using `{% raw}` to avoid rendering doesn't work in my case because I render the files twice. With `{% raw}`, the second render doesn't work.

Reply:
Doesn't hurt much, as it's scoped to your tool anyway. Just for my own curiosity: the `{% raw %}` tags would be gone after the first render pass, no? Or am I missing something?

Reply:
yes, it will go after the first pass. I need two pass.